x-pack/plugins/apm/public/components/alerting/transaction_duration_alert_trigger/index.tsx

```diff
@@ -90,6 +91,8 @@ export function TransactionDurationAlertTrigger(props: Props) {
       threshold: 1500,
       windowSize: 5,
       windowUnit: 'm',
+      environment: ENVIRONMENT_ALL.value,
+      transactionType: transactionTypes[0],
     }
   );
```

Comment:
We could make it clear to users that they cannot create alerts from here:

```suggestion
  );
  
  if(!metadata?.serviceName) {
  	return <EuiCallout>Creating alerts from here is not supported. Please go to the [APM app](link))</EuiCallout>
  }
```

Reply:
Mentioned in https://github.com/elastic/kibana/issues/107940

Reply:
Yes, I saw. I was hoping we could do something for 7.15 - like adding a callout that it doesn't work.
Implementing the free form field with suggestion is likely a bigger task and not something we'll get around to before FF.